force-app/main/default/objects/Test_Run_Method_Result__c/fields/New_Failure__c.field-meta.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<CustomField xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>New_Failure__c</fullName>
```

Comment:
this could actually just be a formula just checking if `ISBLANK(First_Failure__c)`?

Reply:
however, if we do it that way, you may not be able to use it in the rollup for the Test_Run